README.md

```diff
@@ -291,6 +291,116 @@ do this on [my personal site](https://github.com/kentcdodds/kentcdodds.com)
 which is generated at build-time on the server. Learn about rendering
 [`react` on the server][react-ssr] and [`glamor` too][glamor-ssr].
 
+### Example Style Objects
+
+Style objects can affect pseudo-classes and pseduo-elements, complex CSS
+selectors, introduce keyframe animations, and use media queries:
+
+<details>
+  <summary>pseudo-class</summary>
+
+  ```javascript
```

Comment:
Ah! Sorry, I included the ```javascript when I said "codeblock". Really sorry about that. Could you deindent that bit as well (looks like this otherwise):

<img width="216" alt="screen shot 2017-04-05 at 5 03 26 pm" src="https://cloud.githubusercontent.com/assets/1500684/24730607/cc4368ea-1a21-11e7-9a32-5acbf275a514.png">

Then I think it'll be perfect! Thanks!